integrated_channels/blackboard/migrations/0001_initial_squashed_0011_alter_blackboardlearnerassessmentdatatransmissionaudit_enterprise_course_enrollment_id.py

```diff
@@ -0,0 +1,124 @@
+# Generated by Django 3.2.11 on 2022-01-27 11:11
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.migrations.operations.special
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+import simple_history.models
+import uuid
+
+
+def create_uuid(apps, schema_editor):
+    BlackboardEnterpriseCustomerConfiguration = apps.get_model('blackboard', 'BlackboardEnterpriseCustomerConfiguration')
+    for config in BlackboardEnterpriseCustomerConfiguration.objects.all():
+        config.uuid = uuid.uuid4()
+        config.save()
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('blackboard', '0001_initial'), ('blackboard', '0002_auto_20200930_1723'), ('blackboard', '0003_blackboardlearnerdatatransmissionaudit'), ('blackboard', '0004_blackboard_tx_chunk_size_default_1'), ('blackboard', '0005_blackboardlearnerassessmentdatatransmissionaudit'), ('blackboard', '0006_auto_20210708_1446'), ('blackboard', '0007_auto_20210909_1536'), ('blackboard', '0008_auto_20210923_1727'), ('blackboard', '0009_alter_blackboardenterprisecustomerconfiguration_enterprise_customer'), ('blackboard', '0010_auto_20211221_1532'), ('blackboard', '0011_alter_blackboardlearnerassessmentdatatransmissionaudit_enterprise_course_enrollment_id')]
```

Comment:
not sure what this migration is about?

Reply:
This is a squash migration. You can read more about it here: https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-SquashingMigrations. This is how we are able to get the old migration not to run for new deployments, which was causing errors. For older deployments, just the new AlterField migration will be run.